src/main/java/seedu/address/logic/commands/project/AddTaskCommand.java

```diff
@@ -27,7 +27,6 @@
             + PREFIX_TASK_IS_DONE + "TASK STATUS "
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_PROJECT_NAME + "Create new Person Class "
-            + PREFIX_DEADLINE + "29-02-2020 00:00:00 "
```

Comment:
Why do you remove this line?

Reply:
because deadline is not a compulsory parameter of Task, and will cause error if the deadline is set when instantiating